test/helpers/plugin_server.go

```diff
@@ -22,7 +22,7 @@ func main() {
 		BinariesDirectory: binDir,
 		Server: fake.PluginServer{
 			Host: host,
-			Port: 3000,
+			Port: 3010,
```

Comment:
you can also make it configurable in the same why as we have for `PLUGIN_SERVER_HOST` 👍